prymer/primer3/primer3_parameters.py

```diff
@@ -196,6 +209,9 @@ class ProbeParameters:
     probe_max_dinuc_bases: int = 4
     probe_max_polyX: int = 5
     probe_max_Ns: int = 0
+    probe_max_self_any_thermo: Optional[float] = None
+    probe_max_self_end_thermo: Optional[float] = None
+    probe_max_hairpin_thermo: Optional[float] = None
```

Comment:
Same question here on renaming these.  FWIW while I'm pro renmaing in `Oligo` I'm about 50/50 here as this is much more specific to primer3.

Reply:
I think it adds clarity to have them all be named similarly across `Oligo`, `*Weights` and `*Parameters` -- a user can tell, from the name, what parameters they are tweaking, how much each is going to be weighted, and the corresponding characteristics of the design candidate

Reply:
Seconding Erin here

Reply:
Third ;)